tests/test_checkers.py

```diff
@@ -391,6 +391,18 @@ class DummyDict(TypedDict, total=total):
         else:
             check_type(value, DummyDict)
 
+    @pytest.mark.skipif(
+        sys.version_info < (3, 10), reason="Only required on Python 3.10+"
+    )
```

Comment:
Does it actually hurt if this test is run on older Pythons?

Reply:
No, but it would require `typing_extensions` as an additional test dependency on Python 3.9. I think it isn't required, because the problematic `is_typeddict` only exists since Python 3.10.

Reply:
Yup, I came to the same conclusion and decided to drop the issue.